docs/configuration.rst

```diff
@@ -14,24 +14,30 @@ Running in production
 Recommended settings
 --------------------
 
-Most default setting values in the application code base are suitable for production.
+Most default setting values in the application code base are suitable
+for production.
 
-But the set of settings mentionned below might deserve some review or adjustments:
+Once `PostgreSQL is installed <postgresql-install>`_, the settings about
+backends as shown in :file:`config/kinto.ini` can be uncommented in order
+to use *PostgreSQL*.
 
+Also, the set of settings mentionned below might deserve some review or adjustments:
 
 .. code-block :: ini
 
+    kinto.flush_endpoint_enabled = false
     cliquet.http_scheme = https
     cliquet.paginate_by = 100
     cliquet.batch_max_requests = 25
     cliquet.storage_pool_maxconn = 50
     cliquet.cache_pool_maxconn = 50
+    cliquet.permission_pool_maxconn = 50
     fxa-oauth.cache_ttl_seconds = 3600
 
 :note:
 
     For an exhaustive list of available settings and their default values,
-    refer to `the source code <https://github.com/mozilla-services/cliquet/blob/2.0.0/cliquet/__init__.py#L26-L78>`_.
+    refer to `the source code <https://github.com/mozilla-services/cliquet/blob/2.1.0/cliquet/__init__.py#L26-L78>`_.
```

Comment:
Should we put master here to avoid forgetting to update it? Cliquet 2.2 has been released already so this rotted.